settings/DevHome.Settings/Views/AccountsPage.xaml.cs

```diff
@@ -208,4 +210,25 @@ private async void Logout_Click(object sender, RoutedEventArgs e)
             _ = await afterLogoutContentDialog.ShowAsync();
         }
     }
+
+    private async Task InitiateAddAccountUserExperienceAsync(Page parentPage, AccountsProviderViewModel accountProvider)
+    {
+        if (accountProvider.DeveloperIdProvider.GetAuthenticationExperienceKind() == Microsoft.Windows.DevHome.SDK.AuthenticationExperienceKind.CardSession)

```

Comment:
nit: `using Microsoft.Windows.DevHome.SDK;`

Reply:
Referenced in file. Thanks!

Reply:
With the using, you can drop `Microsoft.Windows.DevHome.SDK.` from `AuthenticationExperienceKind.CardSession.`.